index.html

```diff
@@ -113,7 +157,7 @@ <h3 class="title-project">Project name goes here</h3>
               <li>Ruby on Rails</li>
               <li>JavaScript</li>
             </ul>
-            <button type="button" class="see-project">
+            <button id="button2"   type="button" class="see-project">
               See this project<img src="Images/Union.svg" alt="show-project" />
```

Comment:
- [ ] [__Optional__] I think it would be nice if you work on the buttons hover and pressed effects before merging as shown in the design, currently they are only available on mobile view only. Remember having outstanding UI increases a chance of getting job offers as recruiters check projects.